tests/js/client/shell/transaction/shell-transaction-intermediate-commit-cluster.js

```diff
@@ -165,14 +173,22 @@ function transactionIntermediateCommitsSingleSuite() {
       } catch (err) {
       }
       assertEqual(didWork, false);
-      
-      // follower must have been dropped
-      let droppedFollowersAfter = getMetric(leader, "arangodb_dropped_followers_total");
-      assertEqual(droppedFollowersBefore + 1, droppedFollowersAfter);
+
+      if (!isReplication2) {
+        // follower must have been dropped
+        let droppedFollowersAfter = getMetric(leader, "arangodb_dropped_followers_total");
+        assertEqual(droppedFollowersBefore + 1, droppedFollowersAfter);
+      }
```

Comment:
Should we add the "else" branch, that asserts that droppedFollowers is not changed?

The metric is simply not in use in Rep2 as the concept does not exist.